src/components/TextInput/TextInput.tsx

```diff
@@ -111,7 +111,19 @@ export const TextInput = React.forwardRef((props: Props, ref: ForwardedRef<RNTex
       />
 
       {!noErrors && (
-        <HelperText type={errorTextEnabled && !isEmptyString(errorText) ? 'error' : 'info'} visible>
+        <HelperText
+          theme={{
+            roundness: 8,
+            colors: {
+              background: COLORS.BACKGROUND,
+              placeholder: faded ? COLORS.GREY_6 : COLORS.TEXT_INPUT,
+              primary: faded ? COLORS.GREY_6 : COLORS.BLACK,
+              error: COLORS.ERROR_TEXT_COLOR,
+            },
+          }}
+          type={errorTextEnabled && !isEmptyString(errorText) ? 'error' : 'info'}
+          visible
+        >
```

Comment:
To unify styles

<img width="358" alt="Screenshot 2023-02-07 at 14 41 22" src="https://user-images.githubusercontent.com/105349292/217334206-1c40d575-378e-4aba-8129-087aee15ed2b.png">